### ISSUE TITLE
Base/Env/InitSystem doesn't detect OpenRC for Alpine Linux / Busybox

### ISSUE THREAD

**Issue by uturunku1** [newrelic/newrelic-diagnostics-cli#32]
## Summary

```
Base/Env/InitSystem
Summary
Unable to parse init system from: /bin/busybox
```

https://wiki.gentoo.org/wiki/OpenRC#busybox

In the code: https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/base/env/initSystem.go#L69

**Comment by uturunku1** [comment]
The issue was addressed here: https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/base/env/initSystem.go#L91
